src/bench/cluster_linearize.cpp

```diff
@@ -242,12 +290,84 @@ static void MergeLinearizations64TxWorstCase(benchmark::Bench& bench) { BenchMer
 static void MergeLinearizations75TxWorstCase(benchmark::Bench& bench) { BenchMergeLinearizationsWorstCase<BitSet<75>>(75, bench); }
 static void MergeLinearizations99TxWorstCase(benchmark::Bench& bench) { BenchMergeLinearizationsWorstCase<BitSet<99>>(99, bench); }
 
-BENCHMARK(LinearizePerIter16TxWorstCase, benchmark::PriorityLevel::HIGH);
-BENCHMARK(LinearizePerIter32TxWorstCase, benchmark::PriorityLevel::HIGH);
-BENCHMARK(LinearizePerIter48TxWorstCase, benchmark::PriorityLevel::HIGH);
-BENCHMARK(LinearizePerIter64TxWorstCase, benchmark::PriorityLevel::HIGH);
-BENCHMARK(LinearizePerIter75TxWorstCase, benchmark::PriorityLevel::HIGH);
-BENCHMARK(LinearizePerIter99TxWorstCase, benchmark::PriorityLevel::HIGH);
+// The following example clusters were constructed by replaying historical mempool activity, and
+// selecting for ones that take many iterations (after the introduction of some but not all
+// linearization algorithm optimizations).
+
+/* 2023-05-05T23:12:21Z 71, 521780, 543141,*/
```

Comment:
Are these comments in the format timestamp, tx count, fee, size (or similar)? Or are they actually something like iteration count? I added a print statement to get feerates of clusters but got something slightly different.

```
total feerate: 4818977 / 54585, 71 txns
total feerate: 15029636 / 99008, 81 txns
total feerate: 1288344 / 98145, 90 txns
total feerate: 15755611 / 77992, 87 txns
total feerate: 429736 / 8245, 35 txns
total feerate: 695118 / 60056, 60 txns
total feerate: 941605 / 80052, 99 txns
total feerate: 1948779 / 51103, 52 txns
total feerate: 11583092 / 76049, 69 txns
total feerate: 871078 / 55437, 77 txns
total feerate: 6127495 / 94464, 48 txns
total feerate: 1176444 / 14638, 77 txns
total feerate: 76548 / 7724, 40 txns
total feerate: 10487482 / 100707, 96 txns
total feerate: 14575007 / 90755, 93 txns
total feerate: 867760 / 10558, 55 txns
total feerate: 1205280 / 14663, 76 txns
total feerate: 1493360 / 18235, 98 txns
total feerate: 1446604 / 17766, 99 txns
total feerate: 457656 / 23949, 78 txns
```

Reply:
The format was timestamp (from my simulation), tx count, min iterations when running Linearize 10 times with different rng seeds and then once more when running it woth LIMO using the optimal linearization as input, and then max iterations over the same set of 11 runs.

Reply:
aha, that makes much more sense!

Reply:
would be worth adding that description if touched again or in followup

Reply:
+1 on adding this in a follow-up.
I can create a follow for it see https://github.com/ismaelsadeeq/bitcoin/commit/9350732b17915d315cf3ce9c744d1491c1b5d386 with the description @sdaftuar provided